app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt

```diff
@@ -239,14 +272,6 @@ private val links = listOf(
         R.drawable.ic_link_wheelmap,
         R.string.link_wheelmap_description
     ),
-    Link(
-        "openvegemap",
-        "https://openvegemap.netlib.re",
```

Comment:
Why not just replace it with a link to https://www.veggiekarte.de/ as mentioned here:
https://github.com/Rudloff/openvegemap/issues/134#issuecomment-1516185005

Seems to do the same thing in a similar way.

Reply:
Hmm, good idea actually. Didn't know this existed and must have overread it in case it was mentioned.

Reply:
I mean it still seems a shame to me that someone hasn't just made a generic open diet map which you could toggle between say vegetarian/vegan, halal, gluten free etc, but there we go.

Reply:
done.

Well, I think it makes most sense to do that in mapcomplete?